changelogs/3.0.0.md

```diff
@@ -0,0 +1,478 @@
+  ```lua
+    box.cfg{
+        bootstrap_strategy = 'config',
+        bootstrap_leader = 'leader-name',
+        replication = {
+            ...
+        },
+        ...
+    }
+  ```
+  (gh-7999, gh-8539).
+
+### LuaJIT
+
+* LuaJIT now can handle exceptions on traces (gh-7745).
+
+### Lua
+
+* **[Breaking change]** Added the new `varbinary` type to Lua. An object of
+  this type is similar to a plain string but encoded in MsgPack as `MP_BIN` so
+  it can be used for storing binary blobs in the database. This also works the
+  other way round: data fields stored as `MP_BIN` are now decoded in Lua as
+  varbinary objects, not as plain strings, as they used to be. Since the latter
+  may cause compatibility issues, the new compat option `binary_data_decoding`
+  was introduced to revert the built-in decoder to the old behavior (gh-1629).
+* Introduced the new Lua module `trigger`. This module enables managing and
+  calling triggers stored in trigger registry (gh-8656).
+* Triggers from `space_object`, `box.session`, and `box.ctl` were moved to
+  the trigger registry (gh-8657).
+* Introduced a new event 'tarantool.trigger.on_change' in the trigger registry.
+  It is called when any event in the trigger registry is modified (gh-8664).
+* Introduced helpers for encoding and decoding IPROTO packets in Lua (gh-8054).
+
+#### Msgpack
+
+* Improved error reporting for `msgpack.decode`. Now, an error raised by
+  `mgpack.decode` has a detailed error message and the offset in the input
+  data. If `msgpack.decode` failed to unpack a MsgPack extension, it also
+  includes the error cause pointing to the error in the extension data
+  (gh-7986).
+
+#### Netbox
+
+* Introduced the new `net.box` module function `from_fd` for creating a new
+  connection from a socket file descriptor number (gh-8984).
+
+#### Popen
+
+* Introduced new option `inherit_fds` for `popen.new`. The option takes
+  an array of file descriptor numbers that should be left open in the child
+  process (gh-8926).
+
+#### Socket
+
+* Introduced new socket functions `socket.socketpair`, `socket.from_fd`, and
+  `socket:detach` (gh-8927).
+
+#### Xlog
+
+* Introduced the `xlog.meta()` method for reading a meta block from an xlog
+  file.
+
+### SQL
+
+* Introduced the `SHOW CREATE TABLE` statement (gh-8098).
+* **[Breaking change]** The `DROP CONSTRAINT` statement has been
+  improved (gh-9112).
+* **[Breaking change]** Names in SQL are now case sensitive. To support backward
+  compatibility, a second lookup using a name normalized using the old rules is
+  added (gh-4467).
+* `SHOW CREATE TABLE` no longer supports the DEFAULT clause (gh-8793).
+* `SQL_EXPR` functions can now be set as a default value (gh-8793).
+* A literal set as the default value can no longer have a `+` or `-` sign unless
+  the literal is numeric (gh-8793).
+* **[Breaking change]** SQL now uses the BOX mechanism for default
+  values (gh-8793).
+
+### App
+
+* Added the new `--force-recovery` CLI option (gh-8876).
```

Comment:
This section contains just one item regarding a CLI option. I would merge it into the CLI section.

Reply:
ditto